cartographer_ros/CMakeLists.txt

```diff
@@ -83,9 +83,9 @@ catkin_package(
   LIBRARIES ${PROJECT_NAME}
 )
 
-file(GLOB_RECURSE ALL_SRCS "*.cc" "*.h")
-file(GLOB_RECURSE ALL_TESTS "*_test.cc")
-file(GLOB_RECURSE ALL_EXECUTABLES "*_main.cc")
+file(GLOB_RECURSE ALL_SRCS "cartographer_ros/*.cc" "cartographer_ros/*.h")
+file(GLOB_RECURSE ALL_TESTS "cartographer_ros/*_test.cc")
+file(GLOB_RECURSE ALL_EXECUTABLES "cartographer_ros/*_main.cc")
```

Comment:
The `scripts` folder also has a `.cc` executable.

Reply:
I don't have any `.cc` files in my scripts folder on the current master branch. Are some of these generated?

Reply:
True. I misread something :)